### ISSUE TITLE
how to test

### ISSUE THREAD

**Issue by wangping1408** [yinguobing/cnn-facial-landmark#73]
Hello, I want to know the newly modified landmark.py, how to use the model for testing. The previous code is ready for testing. thank you

**Comment by yinguobing** [comment]
Are you using the 'master' branch?

wangping1408 <notifications@github.com> 于2019年9月2日周一 下午3:19写道：

**Comment by wangping1408** [comment]
@yinguobing Yes, I am using the landmark.py code in this 'master'  branch and trained my model, but I don't know how I would use this code to test the generated model.

**Comment by yinguobing** [comment]
I would recommand using the `keras` branch that makes testing much easier.
